.github/pull_request_template.md

```diff
@@ -1,9 +1,10 @@
-## Why was this change made? 🤔
+# Why was this change made? 🤔
 
 
 
-## How was this change tested? 🤨
+# How was this change tested? 🤨
 
 ⚡ ⚠ If this change involves consuming from or writing to another service (or shared file system), ***run [integration test create_object_h2_spec.rb](https://github.com/sul-dlss/infrastructure-integration-test)*** and/or test manually in [stage|qa] environment, in addition to specs. ⚡
 
+# Does your change introduce accessibility violations (at the WCAG A or AA conformance levels)? 🩺
```

Comment:
consider leaving space to answer this and the previous question.

Reply:
consider:

```
# If your change affects user facing pages, did you check before and after with the siteimprove Accessibility Checker?
```

Reply:
@ndushay I added space, re-phrased, and added a link to the (in-progress) Infra a11y guide. Have another look?

Reply:
great - thanks Mike!